bots/image-download

```diff
@@ -203,14 +203,21 @@ def download_images(image_list, force, quiet, state, store):
                 if os.path.islink(link) and os.path.dirname(os.readlink(link)) == "":
                     image_list.append(filename)
 
-    for image in image_list:
-        if state:
-            target = state_target(image)
-        else:
-            target = committed_target(image)
-        if force or state or not os.path.exists(target):
-            download(target, force, quiet, store)
+    success = True
 
+    for image in image_list:
+        try:
+            if state:
+                target = state_target(image)
+            else:
+                target = committed_target(image)
+            if force or state or not os.path.exists(target):
+                download(target, force, quiet, store)
+        except Exception as ex:
```

Comment:
I can't say I like this overly general `Exception` here - would be better to limit it to e. g. `HTTPError` or `IOError`, or the original `RuntimeError`. Like that it also easily hides variable typos, `SyntaxError`s and whatnot. But CI says that image-download still works, so I won't dwell on it. (Just pointing out for the future).

Reply:
Point taken, but notice that in this case the exception is not swallowed.  Also, the intent is to continue with the next image, no matter what happened when downloading the current one.  So even if we hit a SyntaxError, I'd still want to continue with the next image.